tooling.yml

```diff
@@ -0,0 +1,5 @@
+architecture-sniffer:
+  priority: 1
```

Comment:
must always be 2

also run from project level to see current issues in baseline